dist/server/DeviceServer.js

```diff
@@ -275,7 +275,7 @@ var DeviceServer = function () {
                   deviceID: deviceID,
                   isPublic: isPublic,
                   name: message.getUriPath().substr(3),
-                  ttl: message.getMaxAge() > 0 ? message.getMaxAge() : 60,
+                  ttl: message.getMaxAge() > 0 && message.getMaxAge(),
```

Comment:
I don't think this is correct. You should just replace `0` with the `DEFAULT_EVENT_TTL`